Modules/_multiprocessing/multiprocessing.c

```diff
@@ -139,34 +139,15 @@ static PyMethodDef module_methods[] = {
  * Initialize
  */
 
-static struct PyModuleDef multiprocessing_module = {
-    PyModuleDef_HEAD_INIT,
-    "_multiprocessing",
-    NULL,
-    -1,
-    module_methods,
-    NULL,
-    NULL,
-    NULL,
-    NULL
-};
-
-
-PyMODINIT_FUNC
-PyInit__multiprocessing(void)
+int multiprocessing_exec(PyObject *module)
```

Comment:
Such function must be static:

```suggestion
static int
multiprocessing_exec(PyObject *module)
```

Try to run `make smelly` to check that you don't export functions by mistake.

Reply:
@vstinner Is there a way to run make smelly on windows?  I use WSL when I need to test something on linux but most of the time I use windows